Packages/com.llealloo.audiolink/Runtime/AudioLink.asmdef

```diff
@@ -3,17 +3,16 @@
     "references": [
         "UdonSharp.Runtime",
         "UdonSharp.Lib",
-        "VRC.Udon"
+        "VRC.Udon",
+        "VRC.Udon.Serialization.OdinSerializer"
```

Comment:
why?

Reply:
referencing IUdonEventReceiver causes this otherwise
```
The type 'ISupportsPrefabSerialization' is defined in an assembly that is not referenced. You must add a reference to assembly 'VRC.Udon.Serialization.OdinSerializer, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
```